src/Tribe/Models/Post_Types/Event.php

```diff
@@ -333,4 +335,59 @@ public static function prime_cache( array $events = [] ) {
 
 		_prime_post_caches( $posts );
 	}
+
+	/**
+	 * Overrides the base method to conver the I18n Dates to PHP built-in Date types.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string,mixed> $properties The properties to filter.
+	 *
+	 * @return array<string,mixed> The filtered properties.
+	 *
+	 * @throws \Exception If a date cannot be converted to a DateTime object.
+	 */
+	protected function scalar_serialize_properties( array $properties ): array {
+		// Convert the dates to built-in PHP date objects.
+		$properties['dates'] = array_map( static function ( $date ) {
+			if ( $date instanceof Date_I18n_Immutable ) {
+				return new \DateTimeImmutable( $date->format( 'Y-m-d H:i:s' ), $date->getTimezone() );
+			}
+
+			if ( $date instanceof Date_I18n ) {
+				return new \DateTime( $date->format( 'Y-m-d H:i:s' ), $date->getTimezone() );
+			}
+
+			return $date;
+		}, get_object_vars( $properties['dates'] ) );
+
+		return $properties;
+	}
+
+	/**
+	 * Overrides the base method to convert date properties to I18n Dates.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string,mixed> $properties The properties to filter.
+	 *
+	 * @return array<string,mixed> The filtered properties.
+	 *
+	 * @throws \Exception If a date cannot be converted to a I18n Date object.
+	 */
+	protected function scalar_unserialize_properties( array $properties ): array {
+		$properties['dates'] = (object) array_map( static function ( $date ) {
+			if ( $date instanceof \DateTimeImmutable ) {
+				return new Date_I18n_Immutable( $date->format( 'Y-m-d H:i:s' ), $date->getTimezone() );
+			}
+
+			if ( $date instanceof \DateTime ) {
+				return new Date_I18n( $date->format( 'Y-m-d H:i:s' ), $date->getTimezone() );
+			}
+
+			return $date;
+		}, $properties['dates'] ?? [] );
+
+		return $properties;
+	}
```

Comment:
Where are these two beign used?

Reply:
Left over from another iteration, no longer required, nice catch.

Reply:
They are used in the Common `Base` during serialization for the purpose of object caching, and during unserialization from object caching.